patient-dao/src/main/java/hu/kuncystem/patient/dao/user/JDBCUserGroupDao.java

```diff
@@ -66,31 +68,56 @@ public UserGroup mapRow(ResultSet rs, int row) throws SQLException {
     private JdbcOperations jdbc;
 
     public List<User> getAllUserFromGroup(UserGroup group) {
-        return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        try {
+            return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        } catch (EmptyResultDataAccessException e) {
+            return null;

```

Comment:
This is a general comment and applies to other places in your code: it's better to return an empty collection instead of null for the following reasons:
- you can avoid tons of null checks in your code
- you can avoid NPEs being thrown if you failed to check for null